File: Python Code/Daily_Flash_Code_0/Week9/21Mar/53-DailyFlash_Solutions/21_March_Solution_Three/C/prog5.c
#include<stdio.h>
#include<math.h>
int main() {
	int n, sum = 0, num;
	scanf("%d", &n);
	int arr[n];
	for(int i = 0; i < n; i++) {
		scanf("%d", &arr[i]);
		sum += arr[i];
	}
	float avg = (float)sum / (float)n;
	sum = 0;
	for(int i = 0; i < n; i++) {
		sum += (arr[i] - avg) * (arr[i] - avg);
	}
	printf("var : %f\nstd dav : %f", (float)sum / n, sqrt((float)sum / n));
}

File: Python Code/Daily_Flash_Code_0/Week9/21Mar/53-DailyFlash_Solutions/21_March_Solution_Three/Python/prog5.py
import array
import math
num = int(input())
s = 0
arr = array.array("i", []);
for i in range(num):
    n = int(input())
    s += n
    arr.append(n);
    
avg = s / len(arr)
s = 0
for i in arr:
    s += (avg - i) * (avg - i)
print(s / len(arr))
print(math.sqrt(s / len(arr)))
